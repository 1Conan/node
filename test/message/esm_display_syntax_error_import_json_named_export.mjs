// Flags: --experimental-json-modules --no-warnings
/* eslint-disable no-unused-vars */
import '../common/index.mjs';
import { ofLife } from '../fixtures/experimental.json' assert { type: 'json' };
