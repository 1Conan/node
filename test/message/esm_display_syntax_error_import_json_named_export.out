file:///*/test/message/esm_display_syntax_error_import_json_named_export.mjs:*
import { ofLife } from '../fixtures/experimental.json' assert { type: 'json' };
         ^^^^^^
SyntaxError: The requested module '../fixtures/experimental.json' does not provide an export named 'ofLife'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:*:*)
    at async ModuleJob.run (node:internal/modules/esm/module_job:*:*)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:*:*)
    at async loadESM (node:internal/process/esm_loader:*:*)
    at async handleMainPromise (node:internal/modules/run_main:*:*)

Node.js *
